File: src/main/java/no/nav/pensjon/innsyn/domain/desc/FppAfpDescriptor.java
package no.nav.pensjon.innsyn.domain.desc;

import no.nav.pensjon.innsyn.entity.EntityDescriptor;

public class FppAfpDescriptor implements EntityDescriptor {

    private static final String ENTITY_NAME = "FPP AFP";

    private static final String[] PROPERTY_NAMES = {
            "Status",
            "Framtidige pensjonspoeng AFP",
            "Gjelder f.o.m.",
            "Gjelder t.o.m.",
            "AFP pensjonsgrad",
            "AFP type"
    };

    public String getEntityName() {
        return ENTITY_NAME;
    }

    public String[] getPropertyNames() {
        return PROPERTY_NAMES;
    }
}

File: src/main/kotlin/no/nav/pensjon/innsyn/domain/desc/FppAfpDescriptor.kt
package no.nav.pensjon.innsyn.domain.desc

import no.nav.pensjon.innsyn.domain.FppAfp
import no.nav.pensjon.innsyn.entity.EntityDescriptor

object FppAfpDescriptor : EntityDescriptor<FppAfp>("FPP AFP",
        arrayOf(
                "Status",
                "Framtidige pensjonspoeng AFP",
                "Gjelder f.o.m.",
                "Gjelder t.o.m.",
                "AFP pensjonsgrad",
                "AFP type"
        )
)